tools/cvm-image-rewriter/plugins/08-ccnp-uds-directory-permission/README.md

```diff
@@ -0,0 +1,3 @@
+# CCNP UDS directory permission
+
+This plugin is used to prepare shared Unix domain socket directories for CCNP. Users do not need to customize.
```

Comment:
I would recommend the same language as above to indicate they don't need to do this but then give the reason they might.